<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DFS_BFS_Widget</class>
 <widget class="QWidget" name="DFS_BFS_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1302</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="IODataBox">
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="title">
      <string>Nạp đồ thị</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="algorithmsLabel">
        <property name="text">
         <string>Chọn thuật toán</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="loadFileButton">
        <property name="text">
         <string>Nạp file dữ liệu</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="dataStructuresLabel">
        <property name="text">
         <string>Cấu trúc dữ liệu</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="adjacenyMatrixRB">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Ma trận kề</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">dataStructures</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QRadioButton" name="listRB">
        <property name="text">
         <string>Danh sách cạnh/cung</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">dataStructures</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>182</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="graphTypesLabel">
        <property name="text">
         <string>Dạng đồ thị</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QRadioButton" name="undirectedRB">
        <property name="text">
         <string>Vô hướng</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">graphTypes</string>
        </attribute>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QRadioButton" name="directedRB">
        <property name="text">
         <string>Có hướng</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">graphTypes</string>
        </attribute>
       </widget>
      </item>
      <item row="3" column="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>182</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QComboBox" name="algorithmsBox">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Thuật toán DFS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Thuật toán BFS</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QLineEdit" name="filePathLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ProcessDataBox">
     <property name="title">
      <string>Thao tác với đồ thị và thuật toán</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0" columnstretch="1,6">
      <item row="1" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>100</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1" rowspan="2">
       <widget class="QTextBrowser" name="solutionView"/>
      </item>
      <item row="0" column="0">
       <widget class="QToolBox" name="processToolsBox">
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="infoTools">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>189</width>
           <height>172</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Ý tưởng</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="analysisButton">
            <property name="text">
             <string>Phân tích - Đánh giá</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sourceCodeButton">
            <property name="text">
             <string>Source Code</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="algorithmsTools">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>207</width>
           <height>176</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Ứng dụng</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="travelButton">
            <property name="text">
             <string>Duyệt đồ thị</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="connectedComButton">
            <property name="text">
             <string>Thành phần liên thông</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="findWayButton">
            <property name="text">
             <string>Tìm đường giữa 2 đỉnh</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="simplifyButton">
            <property name="text">
             <string>Cây khung của đồ thị</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="dataStructures"/>
  <buttongroup name="graphTypes"/>
 </buttongroups>
</ui>
